### DISCUSSION TITLE
Not getting notifications for a new Discussion if converted from an Issue

### DISCUSSION THREAD

**Discussion by wongjn** [community/community#36377] [Discussions]
I have a custom watch settings on a repository to watch on discussions, but discussions created by converting from an issue do not seem to push out a notification for the initial conversion. Subsequent comments *do* send out notifications however.

**Comment by ossanna16** [comment]
Hey there @wongjn and thank you for reporting this! I'm glad you found our community 🙂

**Comment by thomaslevesque** [comment]
I noticed this today as well, but it seems to happen only for the first comment after the issue is converted. I don't receive a notification for the first comment, but I do receive it for subsequent comments. Definitely looks like a bug.

@ossanna16 any news on this?
